short_term/seed.py

```diff
@@ -0,0 +1,93 @@
+"""Script to seed the database with initial metadata values."""
+
+from os import environ as ENV
+
+import pandas as pd
+
+from dotenv import load_dotenv
+import boto3
+import pymssql
+
+from database_handler import get_connection
+
+
+def download_metadata():
+    """Downloads the botanists, plants, and locations metadata from the
+    S3 bucket for seeding."""
+    load_dotenv()
+    s3 = boto3.client('s3')
+    metadata_path = 'metadata/'
+    file_names = ['botanists.csv', 'locations.csv', 'plants.csv']
+    for file_name in file_names:
+        s3_file_path = f"{metadata_path}{file_name}"
+        local_file_path = f"./{file_name}"
+        s3.download_file(ENV['BUCKET_NAME'], s3_file_path, local_file_path)
+
+
+def upload_locations():
+    """Uploads the locations metadata to the database."""
+    locations_df = pd.read_csv('locations.csv')
+    locations_df = locations_df.dropna()
+    locations_tuple = [(row['latitude'], row['longitude'], row['town'],
+                        row['timezone']) for _, row in locations_df.iterrows()]
+
+    with get_connection() as conn:
+        cursor = conn.cursor()
+        insert_query = """INSERT INTO beta.origin_location
+        (latitude, longitude, town, timezone)
+        VALUES (%s, %s, %s, %s)"""
+        cursor.executemany(insert_query, locations_tuple)
+        conn.commit()
+
+
+def upload_plants():
+    """Uploads the plants data into the database."""
+    plants_df = pd.read_csv('plants.csv')
+    plants_df = plants_df.dropna(subset=['plant_name'])
+    plants_df = plants_df.where(pd.notna(plants_df), None)
+
+    locations = get_locations()
+    tuples = [format_plant_tuple(row, locations)
+              for _, row in plants_df.iterrows()]
+
+    with get_connection() as conn:
+        cursor = conn.cursor()
+        insert_query = """INSERT INTO beta.plant
+        (plant_id, plant_name, plant_scientific_name, origin_location_id,
+        small_url, medium_url, original_url, regular_url, thumbnail_url)
+        VALUES (%s, %s, %s, %s, %s, %s, %s, %s, %s)"""
+
+        for plant in tuples:
```

Comment:
Similar case here you can do bulk insert using string manipulation, but not as important since seeding is not time sensitive.